bbq/compiler/compiler.go

```diff
@@ -2286,22 +2289,6 @@ func (c *Compiler[_, T]) addCompiledType(ty sema.Type, data T) uint16 {
 	return uint16(count)
 }
 
-func (c *Compiler[_, _]) enclosingCompositeTypeFullyQualifiedName() string {
```

Comment:
Replaced this manual construction of the qualified-name with `commons.TypeQualifiedName` which relies on types resolved by the checker, which is also used by the VM. This ensures the qualified-names are in sync between the compiler and the vm.